AFMReader/general_loader.py

```diff
@@ -0,0 +1,92 @@
+"""Switchboard for input files."""
+
+from pathlib import Path
+
+import numpy.typing as npt
+
+from AFMReader import asd, gwy, ibw, jpk, spm, stp, top, topostats
+from AFMReader.logging import logger
+
+logger.enable(__package__)
+
+
+# pylint: disable=too-few-public-methods
+class LoadFile:
+    """
+    Class to handle the general loading of an AFM file.
+
+    Parameters
+    ----------
+        filepath : Path
+            Path to the AFM image.
+        channel : str
+            Channel to extract from the AFM image.
+    """
+
+    def __init__(self, filepath: Path, channel: str):
```

Comment:
It adds an onus on the user to ensure `filepath: Path` if we say it can also be `str` then we can easily convert that on instantiation and it doesn't matter if its already `Path`.

```suggestion
    def __init__(self, filepath: str | Path, channel: str):
```